images/ci-kubernetes-e2e-kubeadm/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.4
+VERSION = 0.5
```

Comment:
Can we please switch this to use the same STAMP-COMMIT pattern as everything else?